cpp/velox/compute/WholeStageResultIterator.cc

```
@@ -192,10 +192,13 @@ std::shared_ptr<velox::core::QueryCtx> WholeStageResultIterator::createNewVeloxQ
 
 std::shared_ptr<ColumnarBatch> WholeStageResultIterator::next() {
   tryAddSplitsToTask();
+  if (task_ == nullptr) {
```

Can we throw exception instead of returning nullptr?